src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -745,12 +752,74 @@ void grpc_subchannel_call_unref(
   GRPC_CALL_STACK_UNREF(SUBCHANNEL_CALL_TO_CALL_STACK(c), REF_REASON);
 }
 
+// Sets *status and *server_pushback_md based on md_batch and error.
```

Comment:
This comment is out of date -- there's no `server_pushback_md` parameter here.

Reply:
Done